src/AuthenticationPortal.Contracts/Models/Token.cs

```diff
@@ -0,0 +1,8 @@
+﻿
+namespace AuthenticationPortal.Contracts
+{
+    public class Token
+    {
+        public string TokenString { get; set; }
```

Comment:
do not prefix the property name with datatype

Reply:
Done renamed it to TokenKey